src/components/Sidebar/BranchTransition.tsx

```diff
@@ -28,10 +33,34 @@ const BranchTransition: FC<BranchTransitionProps> = ({
   const transitionKey = transition?.transition || '';
   const transitionNode = pathway.states[transitionKey];
   const [useCriteriaSelected, setUseCriteriaSelected] = useState<boolean>(false);
+  const criteriaDescription = transition.condition?.description;
+  const criteriaIsValid = criteriaDescription != null;
+  const criteriaDisplayIsValid = criteriaDescription && criteriaDescription !== '';
+
+  const buttonText =
+    transition.condition?.cql || transition.condition?.description ? 'DELETE' : 'CANCEL';
+  const icon =
+    transition.condition?.cql || transition.condition?.description ? (
+      <FontAwesomeIcon icon={faTrashAlt} />
+    ) : null;
 
   const handleUseCriteria = useCallback((): void => {
-    setUseCriteriaSelected(!useCriteriaSelected);
-  }, [useCriteriaSelected]);
+    if (transition.condition?.cql || transition.condition?.description) {
+      // delete the the transition
+      if (!currentNodeKey || !transition.id) return;
+      updatePathway(removeTransitionCondition(pathway, currentNodeKey, transition.id));
+      setUseCriteriaSelected(false);
+    } else {
+      setUseCriteriaSelected(!useCriteriaSelected);
+    }
+  }, [
+    useCriteriaSelected,
+    currentNodeKey,
+    pathway,
+    transition.condition,
+    transition.id,
+    updatePathway
+  ]);
```

Comment:
Since `currentNodeKey` will never be null, you can remove the check and simplify this function a bit.
```js
 const handleUseCriteria = useCallback((): void => {
    if (hasCriteria && transition.id) {
      // delete the transition criteria
      updatePathway(removeTransitionCondition(pathway, currentNodeKey, transition.id));
      setUseCriteriaSelected(false);
    } else {
      setUseCriteriaSelected(!useCriteriaSelected);
    }
  }, [useCriteriaSelected, pathway, currentNodeKey, transition.id, updatePathway, hasCriteria]);
```